java/src/jmri/util/swing/JmriPanel.java

```diff
@@ -78,6 +80,10 @@ public List<JMenu> getMenus() {
         return new ArrayList<>();
     }
 
+    public Dimension getMinimumDimension() {
+        return new Dimension(10, 10);
+    }
```

Comment:
What's the purpose of this change?
Does the method returns null if you don't override it?

Reply:
All JmriPanels will have their minimum frame size set to 10 x 10 or approximately 1/4" by 1/4" on a standard monitor which should have no impact on any existing panel.  This is the default. You must override to set a larger minimum.  The reason for the minimums is to prevent small convoluted panels that can't be used or read.